compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/error.emit-freeze-conflicting-global.expect.md

```diff
@@ -0,0 +1,27 @@
+
+## Input
+
+```javascript
+// @enableEmitFreeze @instrumentForget
+function useFoo(props) {
+  const __DEV__ = 'conflicting global';
+  console.log(__DEV__);
+  return foo(props.x);
+}
+
+```
+
+
+## Error
+
+```
+  1 | // @enableEmitFreeze @instrumentForget
+  2 | function useFoo(props) {
+> 3 |   const __DEV__ = 'conflicting global';
+    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Todo: Encountered conflicting global in generated program. Conflict from local binding __DEV__ (3:3)
```

Comment:
why is this one an error rather than renaming?

Reply:
This one is trickier -- we could synthesize a read off of `globalThis` or something similar, but I'm not familiar with browser / node version support. Note that this is a normal bailout (not a build failure)